lesson_01/RandyCastro/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Randy's README</title>
+<link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+
+<h1>Hello my name is Randy💫</h1>
+
+<p>Hello, I'm Randy Castro, and for the past six years, I've been deeply immersed in the dynamic world of retail management, specifically at Banana Republic. However I am now looking to transition into the tech industry. My ultimate goal is to keep on learning new things and branch out to multiple streams of income and retire early.</p>
+
+<h2>What's the best way to reach me? 📞</h2>
+
+<ol>
+  <li>
+    <p>Email <a href="mailto:rcastro@thecodesociety.onmicrosoft.com">rcastro@thecodesociety.onmicrosoft.com</a>: For any business related questions</p>
+  </li>
+  <li>
+    <p>Microsoft Team: For quick questions, updates or friendly messages</p>
+  </li>
+  <li>
+    <p>Text message: For any urgent messages</p>
+  </li>
+</ol>
+
+<img src="https://media.giphy.com/media/IxLeSDtUaZRmSiyCTf/giphy.gif" alt="One eternity later">
+
+<h2>My work hours: 🗓️</h2>
+
+<p>I prefer to kickstart my day with a productive morning routine. Typically, I hit the gym around 7 am to get energized. Then, from 9 am to 5 pm, I focus on tackling my tasks and projects with full dedication and efficiency.</p>
+
+<h2>One of my goals for this year🏆</h2>
+
+<p>My goal is to successfully complete this program and secure a fulfilling career job in the tech industry. I aim to leverage this opportunity and eventually explore other avenues for personal and professional growth.</p>
+
+<h2>Family and Pets</h2>
+
+<p>In my household it is just my wife and our cat name Nori</p>
+
```

Comment:
There are duplicated tags. There should only be one of each of the following tags: <html/>, <head/>, <body/>, and <title/>. Please adjust.